ansible/roles/postldap/defaults/main.yml

```diff
@@ -378,7 +395,9 @@ postldap__postfix__dependent_lookup_tables:
                       )
                     )
                   )'
```

Comment:
The query_filter attribute of the ldap_smtpd_sender_login_maps.cf lookup table can be deduplicated a little bit. The authorizedService matches are in there twice right now. I propose this filter:
```
query_filter: '(&
                 (|
                   (&
                     (objectClass=mailRecipient)
                     (|
                       (mailAddress=%s)
                       (mailAlternateAddress=%s)
                     ) 
                   ) 
                   (&
                     (objectClass=account)
                     (uid=%u)
                     (host=%d)
                   ) 
                 )
                 (|
                   (authorizedService=all)
                   (authorizedService=mail:send)
                 )
               )'
```